docs/static/dead-letter-queues.asciidoc

```diff
@@ -61,16 +61,13 @@ path.dead_letter_queue: "path/to/data/dead_letter_queue"
 NOTE: You may not use the same `dead_letter_queue` path for two different
 Logstash instances.
 
-===== File Rotation
+[[dlq-disk-usage]]
+===== Disk Usage
 
-Dead letter queues have a built-in file rotation policy that manages the file
-size of the queue. When the file size reaches a preconfigured threshold, a new
-file is created automatically.
-
-By default, the maximum size of each dead letter queue is set to 1024mb. To
-change this setting, use the `dead_letter_queue.max_bytes` option.  Entries
-will be dropped if they would increase the size of the dead letter queue beyond
-this setting. 
+Dead letter queues manage their own disk usage, and will not consume more than
+the value of the `dead_letter_queue.max_bytes` option, which defaults to `1024mb`.
+Entries will be dropped if they would increase the size of the dead letter queue
```

Comment:
"Entries will be dropped" < this was recently misinterpreted by a user to mean "old entries" will be discarded, not the new ones, like in a FIFO queue. He wasn't aware that the queue will stop accepting any new entries at this point and was asking us about the error he was getting. Would be great to clarify as well.